src/commands/__test__/tarea.test.ts

```diff
@@ -0,0 +1,316 @@
+        firstName: usersInfoResponse.user.profile.first_name,
+        lastName: usersInfoResponse.user.profile.last_name,
+        email: usersInfoResponse.user.profile.email,
+      });
+
+      expect(commandMock.say).toHaveBeenCalledWith(
+        `Tarea subida con éxito <@${usersInfoResponse.user.id}>!\n\nTarea:\n${fullMessage}\n\n*Para agregar correcciones responder en este hilo.*`
+      );
+
+      expect(threadApi.create).toHaveBeenCalledTimes(1);
+      expect(threadApi.create).toHaveBeenCalledWith({
+        authorId: env.BOT_ID,
+        text: chatPostMessageResponse.message.text,
+        timestamp: chatPostMessageResponse.ts,
+      });
+    });
+  });
+
+  describe('Validations', () => {
+    it('should throw an error when submission has invalid format', async () => {
+      commandMock.command = {
+        text: "Hola dejo la tarea\n\n`console.log('hola')`",
+        channel_name: 'clase-1',
+        user_id: 'mockId',
+      };
+
+      commandMock.client.users.info.mockResolvedValueOnce(
+        // @ts-ignore
+        usersInfoResponse
+      );
+
+      await tareaCommandFunction(
+        // @ts-ignore
+        commandMock
+      );
+
+      expect(uploadTareaMock).toBeCalledTimes(0);
+
+      expect(commandMock.respond).toHaveBeenCalledTimes(1);
+      expect(commandMock.respond).toHaveBeenCalledWith(wrongFormatSubmission());
+    });
+
+    it('should throw an error when trying to make a submission with multiple formats', async () => {
+      commandMock.command = {
+        text: "hola dejo la tarea\n\n```console.log('Hello World!')```\n\ny acá dejo la segunda parte https://github.com/mock/repo",
+        channel_name: 'clase-1',
+        user_id: 'mockId',
+      };
+
+      commandMock.client.users.info.mockResolvedValueOnce(
+        // @ts-ignore
+        usersInfoResponse
+      );
+
+      await tareaCommandFunction(
+        // @ts-ignore
+        commandMock
+      );
+
+      expect(uploadTareaMock).toBeCalledTimes(0);
+
+      expect(commandMock.respond).toHaveBeenCalledTimes(1);
+      expect(commandMock.respond).toHaveBeenCalledWith(
+        multipleSubmissionFormatsDetected()
+      );
+    });
+
+    it('should throw an error when trying to submit a GitHub link in class 4 or less', async () => {
+      commandMock.command = {
+        text: 'Hola acá les dejo la tarea https://github.com/mock/repo',
+        channel_name: 'clase-3',
+        user_id: 'mockId',
+      };
+
+      commandMock.client.users.info.mockResolvedValueOnce(
+        // @ts-ignore
+        usersInfoResponse
+      );
+
+      await tareaCommandFunction(
+        // @ts-ignore
+        commandMock
+      );
+
+      expect(uploadTareaMock).toBeCalledTimes(0);
+
+      expect(commandMock.respond).toHaveBeenCalledTimes(1);
+      expect(commandMock.respond).toHaveBeenCalledWith(wrongFormatBlock());
+    });
+
+    it('should responde with a block when trying to submit a block of code in class 5 or more', async () => {

```

Comment:
should RESPOND with a block when trying to submit a block of code in class 5 or HIGHER.